src/client/styles/components/SubjectHeadings.scss

```diff
@@ -470,3 +471,60 @@ span > .verticalEllipse {
     border-radius: 0.2rem 0 0 0.2rem;
   }
 }
+
+@media screen and (max-width: 757px) and (min-width: 420px) {
+  .alphabeticalPagination {
+    flex-wrap: wrap;
+    a {
+      flex: 6.5%;
+      max-width: 45px;
+    }
+    a:first-child {
+      border-radius: 0.2rem 0 0 0;
+    }
+    a:last-child {
+      border-radius: 0 0 0.2rem 0;
+    }
+    a:nth-child(14) {
+      border-right: 0.08333rem solid #d7d4d0;
+      border-radius: 0 0.2rem 0 0;
+    }
+    a:nth-child(15) {
+      border-radius: 0 0 0 0.2rem;
+    }
+    a:nth-child(-n+14) {
+      border-bottom: unset;
```

Comment:
Looks like 0 0.2rem 0 0 and 0 0 0 0.2rem could be variables.
Also border-radius: (option1), border-radius: (option2), border-bottom: unset could be modules (wouldn't really save anything right now, but it looks like what we really want to say is that for media type 1, the 14th and 15th child should have a certain kind of styling, and for media type 2, child 10 and 21 should have the same styling, so refactoring would reflect that and support more complicated styling if it becomes desirable).